src/compiler/transformer.ts

```diff
@@ -562,31 +562,31 @@ namespace ts {
     }
 
     export const nullTransformationContext: TransformationContext = {
-        get factory() { return factory; },

```

Comment:
This is simply a reordering of existing properties to match the order in `transformNodes` so they (hopefully) share the same "map" in V8.